src/api/mtgql-ep/aliasLibs.ts

```diff
@@ -11,8 +13,12 @@ export const stdlib = (now: Date) => ({
     name: "pauper-commander",
     query: '(is:commander or t:creature) r:u'
   },
+  traditional: {
+    name: "default-skin",
+    hovertext: 'Attempts to filter out digital cards, joke cards, and unreadable showcase printings.',
```

Comment:
oh, that thing, where when you're wondering what a thing is, an you hover your mouse over it, it says more info! Because in this case i don't think the alias is very clear
![image](https://github.com/user-attachments/assets/0b1f130b-1528-41de-a6a8-ae775f1e94b6)